.dockerignore

```diff
@@ -0,0 +1,11 @@
+.git/
+.gitattributes
+.gitignore
+.travis.yml
+changelog.md
+LICENSE
+README*
+THIRDPARTY
+Vagrantfile
+frontend_tests/
+docker-compose*.yml
```

Comment:
note: the file can be autogenerated with https://github.com/jamestalmage/dockerignore, which reduces human err when doing the maintenance.